/**:
  ros__parameters:
    WP1_lat: 37.405071
    WP1_lon: 126.613924
    WP2_lat: 37.403633
    WP2_lon: 126.616342
    WP3_lat: 37.403001
    WP3_lon: 126.614233
    WP4_lat: 37.404870
    WP4_lon: 126.615856
